lumicks/pylake/kymotracker/tests/test_derived_quantities/test_msd.py

```diff
@@ -2,8 +2,11 @@
 import contextlib
 from lumicks.pylake.kymotracker.detail.msd_estimation import *
 from lumicks.pylake.kymotracker.detail.msd_estimation import (
+    _var_cve_known_var,
+    _var_cve_unknown_var,
     _msd_diffusion_covariance,
     _diffusion_ols,
+    _cve,
 )
```

Comment:
hmm, why do we have import all and then import specific functions?

Reply:
Private functions are not imported when importing `*` unless explicitly listed in `__all__`.